justfile

```diff
@@ -26,3 +26,64 @@ inner-wit-publish config-arg:
 
 cargo-check:
     cd packages/wavs-wasi-utils && cargo check --all-targets --all-features
+
+# Update version in all necessary files (eg. just set-tag v0.6.0-alpha.7)
+set-tag version:
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    # Ensure version doesn't start with 'v' for file updates
+    VERSION="{{version}}"
+    if [[ "$VERSION" == v* ]]; then
+        VERSION="${VERSION#v}"
+    fi
+
+    echo "Setting version to: ${VERSION}"
+
+    # Cargo.toml
+    sed -i "s/^version = \".*\"/version = \"${VERSION}\"/" Cargo.toml
+
+    # all WIT packages
+    find wit-definitions -name "*.wit" -type f | while read -r file; do
+        sed -i "s/^package wavs:\([^@]*\)@.*/package wavs:\1@${VERSION};/" "$file"
+        sed -i "s|use wavs:types/\([^@]*\)@[^[:space:]]*|use wavs:types/\1@${VERSION}|g" "$file"
+    done
+
+    echo "Version updated to ${VERSION} in all files"
+
+# Create and push git tags for Rust and go module (eg. just push-tag v0.6.0-alpha.7)
+push-tag version:
```

Comment:
is this called in CI?

Reply:
I made it to be called manually, since it requires some "coordination" with the command above. 
Since `release` job is running already on a pushed tag - so shat you are asking for would be something slightly different, on release to see if given release tag exists with `go/` prefix, if not create it and push it as well.

Reply:
ahh i see, okay that works I get it. thanks